examples/interactions/commands/application_commands.py

```diff
@@ -26,7 +27,7 @@ async def favorite(
     inter,
     anime: pycord.Option = pycord.Option(
         # The type of input the user will put,
-        # for this example it's integer to support autocomplete.
+        # for this example its integer to support autocomplete.
```

Comment:
This is supposed to mean "it is", hence "it's"
```suggestion
        # for this example it's integer to support autocomplete.
```

Reply:
Fixed